recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 test:
     commands:
         - test -f ${PREFIX}/lib/libuuid.a  # [unix]
-        - conda inspect linkages -n _test libuuid  # [unix]
+        - conda inspect linkages libuuid  # [unix]
```

Comment:
It helps us to diagnose issues like: https://github.com/conda-forge/numpy-feedstock/issues/25 but I am not sure how important that is.